fix(ci): update search path for release assets (#125)
jellyfin/jellyfin-apiclient-java: .ci/README.md

## Contribution/Development Note

> Azure pipeline YAMLs can be validated via a Json schema.

Reviewer: ```suggestion
Azure Pipelines YAML files can be validated via a JSON Schema.
```
Both Azure Pipelines and JSON Schema are names so we should use the official capitalization.

Author: Sorry, will change ASAP.

## Contribution/Development Note

Azure pipeline YAMLs can be validated via a Json schema.

> The Azure DevOps documentation mentions this but sends you to get the project scoped schema, which requires authorization.

Reviewer 2: ```suggestion
The Azure DevOps documentation mentions this but asks you to get the project scoped schema, which requires authorization.
```

> A generic schema is although also available and can be helpful to validate:
> 
> https://raw.githubusercontent.com/microsoft/azure-pipelines-vscode/master/service-schema.json

Reviewer 2: ```suggestion
[A generic schema](https://raw.githubusercontent.com/microsoft/azure-pipelines-vscode/master/service-schema.json) is also available however and can be helpful for validation.
```

That should make the bot happy and tweaks the wording a bit.
